Added some missing US President memorial holidays from 1970s and AU/NZ holidays

savvyyabby opened the pull request:
There were some missing US memorial holidays for 1970s.

The ANZAC day treatment in Australia was fixed for some anomalies.

Also added some missing non-trading days for Australia and New Zealand.

See comments in code for details.

jmccorriston commented on README.md:
```diff
@@ -1,3 +1,11 @@
+# jevons global changes
+
+Forked 31-Aug-2019 to fix some holiday omissions for XASX, XNZE and extend all exchanges back to 1-Jan-1970
```
Any chance you could remove these notes from the README?

savvyyabby replied:
Done now! Sorry. forgot about this one.

jmccorriston commented on trading_calendars/exchange_calendar_xasx.py:
```diff
@@ -40,18 +40,28 @@
     observance=weekend_to_monday,
 )
 
+#  Jevons Global: note that prior to 1993 the holiday was observed on the Monday following, or on, the 26th of January
```
I don't know how you were able to get this information from so long ago, but thank you!

savvyyabby replied:
@jmccorriston a range of sources including newspaper checks from an online archive. Let me see if I can codify those. The US presidents holidays came from an NYSE publication. I will go back and find url links where possible.

savvyyabby replied:
On those fixes to tests and the like, give me a day or so to work it out.

jmccorriston replied:
@savvyyabby I'm impressed! I have found it extremely difficult to find corroborating sources prior to ~1990/2000 when it became more common to publish this type of news online.

savvyyabby replied:
@jmccorriston This online archive at the Australian National Library helped me... The Canberra Times published daily stock notices from the Sydney markets. I used that to check some of the more bizarre hiccups in how Australia Day and others were treated over time. It is tricky in Oz because each capital city used to have their own market and they had different holiday dates. That is why these digital newspaper archives are helpful. It is just the whacky stuff, like Australia Day, some ANZAC days and Bicentennial Holidays or specials like Y2K or when NYE is on a Monday .... or something like that. https://trove.nla.gov.au/newspaper/title/11

savvyyabby replied:
I did a bit on India too... there are some good resources from the Reserve Bank of India. I will dig out those too...

savvyyabby replied:
Example of what I mean from newspaper... Sydney stock exchange industrials Jan 5th 1970.

[https://trove.nla.gov.au/newspaper/article/131681315?browse=ndp%3Abrowse%2Ftitle%2FC%2Ftitle%2F11%2F1970%2F01%2F09%2Fpage%2F14540107%2Farticle%2F131681315]

I would just check the week in which a holiday was due to see the observance, like whether the holiday had been mondayized or not.

jmccorriston replied:
Wow, that's awesome!

jmccorriston commented on trading_calendars/exchange_calendar_xnze.py:
```diff
@@ -114,6 +114,20 @@
     start_date="2011-01-01",
 )
 
+# Jevons Global: further ad-hoc holidays and closures
```
It looks like `Timestamp` isn't yet imported in this file, could you add that?

savvyyabby replied:
@jmccorriston sorry for the delay... I just have a rebuild to do

savvyyabby replied:
@jmccorriston added import for Timestamp to trading_calendars/exchange_calendar_xnze.py

savvyyabby commented:
@jmccorriston  All done James!

I fixed the tests for XASX and XNZE by adding/removing the relevant trading days in the test csv files. That fixed most tests, except for the "test_trading_calendar.py" which failed with a special session on the date: 1984-02-01. I could not figure that one out... it looked like it might be Hong Kong, but then I remembered I had changed the default start date to 1970-01-01. Once I reverted the starting date to 1990-01-01 then all the tests passed. Cool. Done-ski.

jmccorriston commented:
Thank you, @savvyyabby, and Happy New Year!
